packages/amplify-e2e-tests/src/cleanup-e2e-resources.ts

```diff
@@ -859,9 +942,18 @@ const cleanup = async (): Promise<void> => {
 
   const filterPredicate = getFilterPredicate(args);
   const accounts = await getAccountsToCleanup();
-
-  await Promise.all(accounts.map((account, i) => cleanupAccount(account, i, filterPredicate)));
+  for(let i = 0 ;i < 5; i ++){
+    console.log("CLEANUP ROUND: ", i + 1);
+    await Promise.all(accounts.map((account, i) => {
+      return cleanupAccount(account, i, filterPredicate);
+    }));
+    await sleep(60 * 1000);// run again after 60 seconds
+  }
   console.log('Done cleaning all accounts!');
 };
 
-void cleanup();
+const sleep = async (ms) => {
```

Comment:
there's already a utility for this in `amplify-e2e-core`

Reply:
thanks, updated